jujugui/static/gui/src/app/state/test-state.js

```diff
@@ -19,6 +19,247 @@ with this program.  If not, see <http://www.gnu.org/licenses/>.
 'use strict';
 
 describe('State', () => {
+
+  const USER_STATE_URLS = [{
```

Comment:
These constants remain the same as were in the previous tests, they were just moved out of the individual tests so that they could be shared between the url parsing and generation suites.